usage/camera/raspicam/timelapse.md

```diff
@@ -38,45 +40,30 @@ Make sure that you use e.g. `%04d` to make `raspistill` output each image to a n
 
-```
-ls *.jpg > stills.txt
-```
-### On the Raspberry Pi
+First you will need to install `ffmpeg`, you may also need to update and upgrade your Pi to successfully install.
 
-Although it will be slow (due to encoding in software rather than using the Raspberry Pi hardware acceleration), you can stitch your JPEG images together using various available tools. This documentation will use `avconv`, which needs to be installed.
-```
-sudo apt install libav-tools
 ```
-Now you can use the tools to convert your JPEG files in to an H264 video file:
+sudo apt update
+sudo apt full-upgrade
+sudo apt install ffmpeg
 ```
-avconv -r 10 -i image%04d.jpg -r 10 -vcodec libx264 -vf scale=1280:720 timelapse.mp4
-```
-On a Raspberry Pi 3, this can encode a little more than one frame per second. The performance of other Pi models will vary. The parameters used are:
-
- - -r 10 Assume ten frames per second in input and output files.
- - -i image%04.jpg The input file specification (to match the files produced during the capture).
- - -vcodec libx264 Use the software x264 encoder.
- - -vf scale=1280:720 Scale to 720p. You can also use 1920:1080, or lower resolutions, depending on your requirements. Note the Pi can only play back up to 1080p video, but if you are intending to play back at, for example, 4K, you could set that here.
- - timelapse.mp4 The name of the output file.
-
-`avconv` has a comprehensive parameter set for varying encoding options and other settings. These can be listed using `avconv --help`.
 
-### On another Linux computer
-
-You can use the same instructions as for the Raspberry Pi, or an alternative package such as `mencoder`:
+Now you can use the `ffmpeg` tool to convert your JPEG files into an mp4 video:
 
 ```
-sudo apt install mencoder
+ffmpeg -r 10 -f image2 -pattern_type glob -i 'image_*.jpg' -s 1280x720 -vcodec libx264 timelapse.mp4
 ```
 
-Now run the following command:
+On a Raspberry Pi 3, this can encode a little more than two frames per second. The performance of other Pi models will vary. The parameters used are:
 
-```
-mencoder -nosound -ovc lavc -lavcopts vcodec=mpeg4:aspect=16/9:vbitrate=8000000 -vf scale=1920:1080 -o timelapse.avi -mf type=jpeg:fps=24 mf://@stills.txt
-```
+- `-r 10` Set frame rate (Hz value) to ten frames per second in the output video.
+- `-f image2` Set ffmpeg to read from a list of image files specified by a pattern.
+- `-pattern_type glob` Select the pattern type used to interpret the provided filenames.
+- `-i 'image_*.jpg'` The input file specification (to match the files produced during the capture).
+- `-s 1280x720` Scale to 720p. You can also use 1920:1080, or lower resolutions, depending on your requirements. Note the Pi can only play back up to 1080p video, but if you are intending to play back at, for example, 4K, you could set that here.
```

Comment:
I suspect the `1920:1080` ought to be `1920x1080` to match the syntax of `1280x720` ?
Also, I believe "the Pi can only play back up to 1080p video" is no longer true for the Pi4?